docs/get-started/query-language-intro.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Introduction to Quickwit's query language
+sidebar_position: 3
+---
+
+Quickwit allows you to search on your indexed documents using a simple query language. Here's a quick overview.
+
+## Clauses
+
+The main concept of this language is a clause, which represents a simple condition that can be tested against documents. 
+
+### Querying fields
+
+A clause operates on fields of your document. It has the following syntax :
+```
+field: condition
```

Comment:
there should be no space between `field:` and `condition`

Reply:
OK